defs/scenarios/storage/ceph/ceph-mon/PG_overdose.yaml

```diff
@@ -0,0 +1,23 @@
+checks:
```

Comment:
can you make the filename all lowercase since thats the convention i.e. ceph-osd/pg_overdose.yaml

Reply:
done